sdk/js/package.json

```diff
@@ -17,7 +17,7 @@
     "build-lib": "tsc -p tsconfig.json && tsc -p tsconfig-cjs.json && node scripts/copyEthersTypes.js",
     "build-all": "npm run build-deps && npm run build-lib",
     "test": "jest --config jestconfig.json --verbose",
-    "test-ci": "NEAR_NO_LOGS=true jest --config jestconfig.json --verbose --setupFiles ./ci-config.js --forceExit",
+    "test-ci": "NEAR_NO_LOGS=true jest --config jestconfig.json --verbose --setupFiles ./ci-config.js --forceExit --runInBand",
```

Comment:
`--runInBand` is going to have a significant performance impact. I would strongly prefer to get all the sdk tests to be async and parallel safe (outside of what is purposefully in a describe block). Understandable as long as this is just set for now to get some better consistency in the tests and removed later.

Reply:
yes, the goal is to get all tests to run reliably first and then we will tackle performance improvements of the tests